scripts/cm_test_all_sandia

```diff
@@ -759,9 +651,7 @@ elif [ "$MACHINE" = "voltrino" ]; then
   BASE_MODULE_LIST="PrgEnv-intel,craype-mic-knl,cmake/3.16.2,slurm/20.11.4a,<COMPILER_NAME>/<COMPILER_VERSION>,gcc/9.3.0"
 
   # Format: (compiler module-list build-list exe-name warning-flag)
-  COMPILERS=("intel/17.0.4 $BASE_MODULE_LIST $INTEL_BUILD_LIST icpc $INTEL_WARNING_FLAGS"
-             "intel/18.0.5 $BASE_MODULE_LIST $INTEL_BUILD_LIST icpc $INTEL_WARNING_FLAGS"
-             "intel/19.0.4 $BASE_MODULE_LIST $INTEL_BUILD_LIST icpc $INTEL_WARNING_FLAGS"
+  COMPILERS=("intel/19.0.4 $BASE_MODULE_LIST $INTEL_BUILD_LIST icpc $INTEL_WARNING_FLAGS"
```

Comment:
Will this still work (in Kokkos develop and 4.0 the new minimum intel version is 19.0.5)?

Reply:
@brian-kelley the test is still compiling+testing with the develop branches with this compiler. At some point it may no longer work, in which case we may need to drop testing on the system with intel (intel/19.0.4 is newest intel compiler, not sure if a newer version will get installed)

Reply:
OK cool, this is fine then.